crates/polars-core/src/testing.rs

```diff
@@ -194,6 +206,19 @@ mod test {
         assert!(df1.equals(&df1))
     }
 
+    #[test]
+    fn assert_df_eq_passes() {
+        let df = df!("a" => [1], "b" => [2]).unwrap();
+        assert_df_eq!(df, df);
+        drop(df); // Ensure `assert_df_eq!` does not consume its arguments.
```

Comment:
Does `assert_eq` take a ref by default?

Reply:
I think so, although it is not entirely obvious to me why the [implementation of `assert_eq`](https://doc.rust-lang.org/src/core/macros/mod.rs.html#36-63) is what it is.